src/requirements/data/colleges/en.ts

```diff
@@ -38,13 +38,14 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Physics',
     description:
-      'PHYS 1112 and 2213, and, depending on the Major, either PHYS 2214 or a designated mathematics or science course.',
+      'The normal program in physics includes PHYS 1112, PHYS 2213, and PHYS 2214 or the corresponding honors courses (PHYS 1116, PHYS 2217, and PHYS 2218).' +
```

Comment:
'PHYS 1112 and 2213, or the corresponding honors courses (PHYS 1116 and PHYS 2217). Depending on the Major, also either PHYS 2214 / PHYS 2218 or a designated mathematics or science course.'

Reply:
sounds good!